services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/allu/CableReportServiceITests.kt

```diff
@@ -384,6 +384,40 @@ class CableReportServiceITests {
         }
     }
 
+    @Nested
+    inner class GetApplicationInformation {
+        @Test
+        fun `returns application information`() {
```

Comment:
This test seems to test more than just what the request returns. It also tests that the request contains correct information. Maybe split that to separate test (or change the name of this test to something like `returns application information with a correct request`?

Reply:
I separated the tests.